tests/modules/nf-core/sgdemux/test.yml

```diff
@@ -0,0 +1,123 @@
+- name: sgdemux test_sgdemux
+  command: nextflow run ./tests/modules/nf-core/sgdemux -entry test_sgdemux -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/sgdemux/nextflow.config
+  tags:
+    - sgdemux
+  files:
+    # fastq's that must exist
+    # md5sum's are unreliable due to lack of sorted output
+    - path: output/sgdemux/output/Undetermined_S25_L001_R1_001.fastq.gz
+      should_exist: true
```

Comment:
```suggestion
```
Let's cut all the `should_exist`s out, they're implicit

Reply:
I see, they have been removed